data/json/items/gun/flintlock.json

```diff
@@ -97,5 +97,14 @@
       [ "stock mount", 1 ]
     ],
     "flags": [ "NEVER_JAMS", "DURABLE_MELEE" ]
+  },
+  {
+    "id": "longrifle_flintlock",
+    "copy-from": "rifle_flintlock",
+    "type": "GUN",
+    "name": "flintlock rifle",
+    "description": "Also called a longrifle, this rifled flintlock gun has signficantly increased accuracy at the cost of taking even longer to reload.",
```

Comment:
I think "Kentucky rifle" or "Pennsylvania rifle" are more common names for this gun in modern America, and a little more distinct than longrifle.

Reply:
Sounds good. Which of the two do you think more common for the general region the game is assumed to take place in?

Reply:
Pennsylvania is closer to New England, maybe that?

Reply:
> Pennsylvania is closer to New England, maybe that?

It's what I'm, thinking, but I don't know if it's one of those things where they call it "Pennsylvania rifle" in Kentucky and "Kentucky rifle" in Pennsylvania, so if someone who's better versed in historical trivia of the American Northeast, I'd appreciate it. If no one is sure, I will just use Pennsylvania for the moment, and change later if anyone has further information.

Reply:
And changed the description to "also called a Pennsylvania rifle", plus a "long barreled" after "this" and before "rifled". If anyone has any information on it not being so, I can change it later.